src/test/java/me/karakelley/http/publicdirectory/PublicDirectoryTest.java

```diff
@@ -76,13 +75,6 @@ void testResourceDoesntExist() {
     assertFalse(publicDirectory.resourceExists("/someBadPath"));
   }
 
-  @Test
-  void testNoFilesInDirectory() {
```

Comment:
I removed this particular test because logically you'd never get to this point in the program if the resource didn't exist (the naming was poor and doesn't express this) now that the validation is all done at a higher level. A test similar to this does exist at a higher level as well.